admin_guide/managing_projects.adoc

```diff
@@ -369,6 +369,77 @@ effect.
 # master-restart controllers
 ----
 
+[[limit-projects-per-service-account]]
+== Enabling and Limiting Self-Provisioned Projects Per Service Account
+
+By default, service accounts cannot create projects. However, administrators can
+enable this capability per service account, and the number of self-provisioned
+projects requested by any given service account can be limited with the
+`ProjectRequestLimit`
+xref:../architecture/additional_concepts/admission_controllers.adoc#architecture-additional-concepts-admission-controllers[admission control plug-in].
+
+[NOTE]
+====
+If service accounts are allowed to create projects, you cannot trust any labels
+placed on them because project editors can manipulate those labels.
+====
+
+. Create a service account in the desired project, if it is does not exist:
++
+----
+$ oc create sa <sa_name>
+----
+
+. As a user with `cluster-admin` privileges, add the `self-provisioner` cluster
+role to the desired service account:
++
+----
+$ oc adm policy \
+    add-cluster-role-to-user self-provisioner \
+    system:serviceaccount:<project>:<sa_name>
+----
+
+. Edit the master configuration file (`/etc/origin/master/master-config.yaml`)
+and set the `maxProjectsForServiceAccounts` field in the `ProjectRequestLimit`
+section to the maximum number of projects any given self-provisioner-enabled
+service account can create.
++
+For example, the following configuration sets a global limit of three projects
+per service account:
++
+[source, yaml]
+----
+admissionConfig:
+  pluginConfig:
+    ProjectRequestLimit:
+      configuration:
+        apiVersion: v1
+        kind: ProjectRequestLimitConfig
+        maxProjectsForServiceAccounts: 3
+----
+
+. After your changes are saved, restart {product-title} for the changes to take
+effect:
++
+----
+# master-restart api
+# master-restart controllers
+----
+
+. Verify that your changes have been applied by logging in as a service account and creating a new project.
+
+.. Log in as the service account using its token:
```

Comment:
s/using/by using
> ! using n
To avoid ambiguity, replace this gerund with either “by using” or “that uses.”